### PR TITLE
Changes for hip-clang for building Caffe2

### PR DESCRIPTION
**Pull request by yxsamliu**

### CODE REVIEW

**Comment by iotamudelta** [comment]
Fixes #11

**Comment by iotamudelta** [comment]
@dagamayank could we get somebody to look at this and, unlike my #11 do something about it? If not, I will review and merge if necessary as this is for PyTorch...

File: cub/device/dispatch/dispatch_histogram.cuh [hunk 1]
```diff
@@ -596,7 +599,6 @@ struct DipatchHistogram
         return CubDebug(hipErrorNotInitialized);
 
     #else
-
```
Comment (iotamudelta) [inline cub/device/dispatch/dispatch_histogram.cuh:599]: whitespace change

**Review by iotamudelta — APPROVED** [review]
Thank you!

**Comment by dagamayank** [comment]
@iotamudelta we can merge this now. can you confirm if the branch it is going into is right?
`hip_port_1.7.4`?

**Comment by iotamudelta** [comment]
@dagamayank that is the default branch, yes. So it should be correct, as that's, e.g., what we are pulling for PyTorch. Merging.

**Pull request merged**